docs/config.md

```diff
@@ -40,7 +42,6 @@ routers:
       commonName: foo
       caCertPath: /foo/caCert.pem
   timeoutMs: 1000
-  dstPrefix: /ext/http
```

Comment:
maybe put this back at the same level as `baseDtab` in the `ext-http` server, and then add `/ext/http => /host/web;` to it's dtab. i think this demonstrates nicely the distinction between an "edge" linkerd and an internal one.